File: Ex3/Task3.5/FieldPerimeter.scala
/**
 * Takes the length and width of a field
 * as command line arguments and prints the perimeter
 * of the field
 */
object FieldPerimeter {
    def main(args: Array[String]) {
        var length: Int = args(0).toInt
        var width: Int = args(1).toInt
        var perimeter: Int = 2 * length + 2 * width

        println("The perimeter of the field is " + perimeter)
    }
}

File: ex3/task3.5/FieldPerimeter.java
/**
* Prints the lenght of the fence needed to enclosure the perimeter of the field
* For testing i have used -20 -30 result was innapropiate
* For 2.1 and 3.1 values the program crashed
* For 20 and 30 values the result was the one expected: 100
* @author Vlad Georgescu	
*/
public class FieldPerimeter
{
    public static void main(String[] args)
    {
        int lenght = Integer.parseInt(args[0]);
        int width = Integer.parseInt(args[1]);
        int perimeter = 2 * lenght + 2 * width;
        System.out.println("The lenght of the fence is " + perimeter);
    }
}